quickstarts/Embeddings.ipynb

```diff
@@ -35,18 +35,34 @@
     },
```

Comment:
<p>(I realise this is out of scope so optional)</p><p> </p><ol><li>This is a long heading.</li><li>The sentence should be "...as to how you'll use..."</li></ol><p> </p><p>For (1) how about something like <code>Specify task_type</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/google-gemini/cookbook/pull/82/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>